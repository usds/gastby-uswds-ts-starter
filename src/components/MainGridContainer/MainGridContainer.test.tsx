import createComponentWithIntl from '@/test/testHelpers';

import MainGridContainer from './MainGridContainer';

describe(`rendering of MainGridContainer Component`, () => {
  const Component = () => <h1>Test</h1>;

  it(`checks if component renders with Full width true`, () => {
    const component = createComponentWithIntl(
      <MainGridContainer fullWidth={true}>
        <Component />
      </MainGridContainer>,
    );

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
  it(`checks if component renders with Full width false`, () => {
    const component = createComponentWithIntl(
      <MainGridContainer fullWidth={false}>
        <Component />
      </MainGridContainer>,
    );

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
