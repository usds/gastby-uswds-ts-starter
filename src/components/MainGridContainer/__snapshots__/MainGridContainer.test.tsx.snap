// Vitest Snapshot v1

exports[`rendering of MainGridContainer Component > checks if component renders with Full width false 1`] = `
<div
  className="grid-container-desktop-lg"
  data-testid="gridContainer"
>
  <h1>
    Test
  </h1>
</div>
`;

exports[`rendering of MainGridContainer Component > checks if component renders with Full width true 1`] = `
<div
  className=""
>
  <h1>
    Test
  </h1>
</div>
`;
